web/package.json

```diff
@@ -46,6 +46,7 @@
     "@opentelemetry/instrumentation": "^0.52.1",
     "@opentelemetry/instrumentation-http": "^0.52.1",
     "@opentelemetry/instrumentation-ioredis": "^0.42.0",
+    "@opentelemetry/instrumentation-winston": "^0.40.0",
```

Comment:
Could this move into the shared package?

Reply:
Probably not as we need to instantiate the instrumentation in the instrumentation files of worker / web. We could do this, if we then expose this to web / dev.